include/FreeRTOS_DNS.h

```diff
@@ -36,6 +36,76 @@
     #include "IPTraceMacroDefaults.h"
 
 
+    #if ( ipconfigBYTE_ORDER == pdFREERTOS_LITTLE_ENDIAN )
```

Comment:
are these defined values that were moved form the source file to the header file going to be used by other files, or it is just a change to accommodate unit testing ?